packages/expo-brightness/src/Brightness.ts

```diff
@@ -57,39 +101,74 @@ export async function setSystemBrightnessAsync(brightnessValue: number): Promise
   return await ExpoBrightness.setSystemBrightnessAsync(clampedBrightnessValue);
 }
 
-export async function useSystemBrightnessAsync(): Promise<void> {
```

Comment:
I think we should keep the old function working at least for one SDK cycle, with the deprecation warning when you use it 🤔

Reply:
Okey, I will revert the rename for now and maybe I will push it as a separate PR, but I will need to add a workaround to the `APISection` component to prevent listing this method as hook (I will need to hardcode the method name in hooks matcher).

Reply:
@tsapeta Oh, unfortunately to fix this properly I will also need the code from `font` PR in `master`.